azurerm/internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -1342,7 +1342,7 @@ func FlattenVirtualMachineScaleSetAutomaticRepairsPolicy(input *compute.Automati
 
 func VirtualMachineScaleSetExtensionsSchema() *schema.Schema {
 	return &schema.Schema{
-		Type:     schema.TypeList,
+		Type:     schema.TypeSet,
```

Comment:
This'll also need the expand functions changed from `.([]interface{})` to `.(*schema.Set).List()`?